src/Analysis/Engine/Impl/DependencyResolution/IAvailableImports.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+namespace Microsoft.PythonTools.Analysis.DependencyResolution {
+    internal interface IAvailableImports {
+        string Name { get; }
```

Comment:
Only `Name`? What about `Modules` and `Packages` in `AvailablePackageImports`?

Reply:
`AvailableModuleImports` doesn't have `Modules` and `Packages`. It only has `ModulePath` which should be used to list variables that are defined in that module (if AST is available).

Reply:
I guess I am confused by plural `Imports` while there is only single `Name`